pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -22,8 +22,9 @@
 from pytorch_lightning.core.step_result import EvalResult, Result
 from pytorch_lightning.loggers import LoggerCollection, TensorBoardLogger
 from pytorch_lightning.trainer.connectors.logger_connector.callback_hook_validator import CallbackHookNameValidator
-from pytorch_lightning.trainer.connectors.logger_connector.epoch_result_store import EpochResultStore, LoggerStages
```

Comment:
do logger stages still exist and if so, are they still needed?

Reply:
Removed !